tests/flux-automation.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+#set -x
+
+_github_head_sha=$1
+_github_base_sha=$2
+
+whitelist_dirs=(
+    admin
+    kube-system
+    kube-public
+    knode-system
+    monitoring
+    neuvector)
+
+exclusions=(
```

Comment:
does this need documentation?

either a comment above or in the error output message?

Reply:
👍